docs/source/json-api/index.rst

```diff
@@ -267,7 +267,7 @@ The JSON API essentially performs two separate tasks:
 Party-specific Access Tokens
 ----------------------------
 
-Party-specific requests, i.e., command submissions and queries, require a JWT with some additional restrictions compared to the format :doc:`described in the Token Payload section here </tools/sandbox>`. For command submissions, ``actAs`` must contain at least one party and ``readAs`` can contain 0 or more parties. Queries require at least one party in either ``actAs`` or ``readAs`` (note that before SDK 1.7.0, every request required exactly one party and before SDK 1.8.0 actAs was limited to exactly one party). In addition to that, the application id and ledger id are mandatory. HTTP requests pass the token in a header, while WebSocket requests pass the token in a subprotocol.
+Party-specific requests, i.e., command submissions and queries, require a JWT with some additional restrictions compared to general :ref:`Daml ledger access token formats <access-token-formats>`. For command submissions, ``actAs`` must contain at least one party and ``readAs`` can contain 0 or more parties. Queries require at least one party in either ``actAs`` or ``readAs`` (note that before SDK 1.7.0, every request required exactly one party and before SDK 1.8.0 actAs was limited to exactly one party). In addition to that, the application id and ledger id are mandatory. HTTP requests pass the token in a header, while WebSocket requests pass the token in a subprotocol.
```

Comment:
"In addition to that" --> "In addtion,"

Reply:
"The application id and ledger id are also mandatory" is even shorter.

Reply:
I would elaborate this paragraph a little more. @stefanobaghino-da maybe you can make a suggestion here or open a ticket to do so later. In short, I think it should be:

When running against a ledger that supports user management, the JSON API accepts any user token and uses that to resolve the bearers rights.

When using custom claims tokens, there is a restriction that at least one party must be set - in `actAs` for submissions, or anywhere for read requests.

Reply:
@bame-da (FYI: @stefanobaghino-da) added your comment to https://github.com/digital-asset/daml/issues/12440

Reply:
@carrielaben-da : addressed your comments in an upcoming commit